src/client/Cargo.toml

```diff
@@ -16,6 +16,7 @@ common-grpc-expr = { path = "../common/grpc-expr" }
 common-query = { path = "../common/query" }
 common-recordbatch = { path = "../common/recordbatch" }
 common-time = { path = "../common/time" }
+common-telemetry = { path = "../common/telemetry" }
```

Comment:
I don't know if it is appropriate to introduce logs here.

Reply:
it's ok